### PR TITLE
Created main_en.md

### PR DESCRIPTION
**Pull request by DomnenkoB**

### CODE REVIEW

File: src/content/main_en.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Manifest · IT Guild Ukraine 
+---
+
+# Manifest
+
+Enough talks, it's time to act!
```
Comment (rozhok) [inline src/content/main_en.md:7]: "Talk is cheap, it's time to act"?

File: src/content/main_en.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+The time has come for IT specialists to unite and protect their rights and interests collectively
+
+Thousands of years people people have united into associations, guilds and trade unions. Some of these organizations are very powerful and have great influence on society. AMPAS awards Oscars, trade union "Solidarity" has played a vital role in freeing Poland of communist rule. Position of Ukrainian IT associations is being taken to notice by deputies of Ukraine.
+
+This is the time for us to create legitimate, independent and democratic professional union - IT specialists Guild
+
+## Aims and goals of the Guild
```
Comment (rozhok) [inline src/content/main_en.md:14]: "Guild goals"

**Comment by rozhok** [comment]
Closing in favor of #5
Thanks for contribution!

**Comment by 1petr0v** [comment]
Hi @DomnenkoB, sorry I haven't seen your PR when assigned to https://github.com/ItGuildUa/discussions/issues/12. I see that we have few differences in the translation, so please feel free to add/amend the translation where you think it proper.

**Pull request closed without merging**